File: python-nodes/linear/02_calculate_subtotal.py
# Node 2: Calculate Subtotal
# Input: order from Node 1  →  Output: order + subtotal

results = []

for item in _items:
    data = item["json"]
    subtotal = round(data["quantity"] * data["unit_price"], 2)
    results.append({"json": {**data, "subtotal": subtotal}})

return results

File: javascript-nodes/linear/02_calculate_subtotal.js
// Node 2: Calculate Subtotal
// Input: order from Node 1  →  Output: order + subtotal

const results = [];

for (const item of $input.all()) {
  const data = item.json;
  const subtotal = Math.round(data.quantity * data.unit_price * 100) / 100;
  results.push({ json: { ...data, subtotal } });
}

return results;
